avm/res/document-db/database-account/tests/e2e/disableLocal/main.test.bicep

```
@@ -0,0 +1,57 @@
+targetScope = 'subscription'
+
+metadata name = 'disable Local'
+metadata description = 'This instance deploys the module disabling local authentication.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-documentdb.databaseaccounts-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'dddadlo'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// Pipeline is selecting random regions which dont support all cosmos features and have constraints when creating new cosmos
+var enforcedLocation = 'eastus'
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2022-09-01' = {
+  name: resourceGroupName
+  location: enforcedLocation
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module disableLocalAuth '../../../main.bicep' = {
```

```suggestion
module testDeployment '../../../main.bicep' = {
```
That's why the script didn't work. As per the specs (a bit hidden maybe) it must be the above name to identify the relevant test 'block'